Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2006-11-09
Initial Package Version: 1.2.2.1
Origin: Jim Gifford
Upstream Status: Unknown
Description: Makes Busybox Friend with Linux Headers

diff -Naur busybox-1.2.2.1.orig/networking/libiproute/iptunnel.c busybox-1.2.2.1/networking/libiproute/iptunnel.c
--- busybox-1.2.2.1.orig/networking/libiproute/iptunnel.c	2006-06-30 15:42:02.000000000 -0700
+++ busybox-1.2.2.1/networking/libiproute/iptunnel.c	2006-11-09 09:24:10.965455917 -0800
@@ -20,10 +20,8 @@
 #include <string.h>
 #include <unistd.h>
 
-#include <netinet/ip.h>
-
-#include <net/if.h>
-#include <net/if_arp.h>
+#include <linux/if.h>
+#include <linux/if_arp.h>
 
 #include <asm/types.h>
 #ifndef __constant_htons
